src/views/TasksDetailsView.vue

```diff
@@ -116,58 +152,52 @@ window.scrollTo({ top: 0, behavior: "smooth" });
+      </v-card>
+      <div class="pointsTemplate__container" @click="handlePointsClick">
+        <v-sheet class="pointsTemplate__top ml-4" :style="{ height: pointsTemplateHeight + 'px' }">
+          {{ taskDetails.points }} <span v-if="showPointsText" style="font-size: 24px;">pkt</span>
+          <v-tooltip activator="parent" location="bottom">Liczba punktów do zdobycia.</v-tooltip>
+        </v-sheet>
+        <div class="pointsTemplate__bottom ml-4"></div>
+      </div>
+    </div>
+    <v-dialog v-model="showDialog" width="290">
+      <v-card class="d-flex flex-column align-center">
+        <v-card-text>
+          Liczba punktów do zdobycia: <br> <b style="display: block; text-align: center;">{{ taskDetails.points }}
+            pkt</b>
+        </v-card-text>
+        <v-btn icon @click="closeDialog" class="mb-4 mt-3">
+          <v-icon>mdi-close</v-icon>
+        </v-btn>
+      </v-card>
+    </v-dialog>
     <v-card class="my-4 px-2 py-2">
       <span class="px-2 text-subtitle-1">{{
         $t("taskDetails.additionalInfo")
       }}</span>
       <v-card-text>{{ taskDetails.additionalDescription }}</v-card-text>
     </v-card>
-    <v-card
-      v-for="(image, attachment) of additionalImages"
-      :key="attachment"
-      class="px-2 py-2 my-2"
-    >
-      <span class="px-2 text-subtitle-2"
-        >{{ $t("taskDetails.attachment") }} {{ attachment + 1 }}</span
-      >
+    <v-card v-for="(image, attachment) of additionalImages" :key="attachment" class="px-2 py-2 my-2">
+      <span class="px-2 text-subtitle-2">{{ $t("taskDetails.attachment") }} {{ attachment + 1 }}</span>
       <img class="h-100 w-100 pt-2" :src="image" />
     </v-card>
-    <user-mini-card
-      :user="createrDetails"
-      :date="taskDetails.creationDate"
-      dateText="taskDetails.taskCreationDate"
-    />
-    <user-mini-card
-      v-if="taskDetails.acceptDate"
-      :user="accepterDetails"
-      :date="taskDetails.acceptDate"
-      dateText="taskDetails.taskAcceptedDate"
-      reverse
-    />
+    <user-mini-card :user="createrDetails" :date="taskDetails.creationDate" dateText="taskDetails.taskCreationDate" />
+    <user-mini-card v-if="taskDetails.acceptDate" :user="accepterDetails" :date="taskDetails.acceptDate"
+      dateText="taskDetails.taskAcceptedDate" reverse />
     <v-card class="my-4 px-2 py-2">
       <span class="px-2 text-subtitle-1">{{
         $t("taskDetails.completedByUsers")
       }}</span>
       <div class="d-flex flex-wrap justify-center px-8 pt-4 ga-2">
-        <btn-with-dropdown
-          v-for="team of teams"
-          :key="team.teamId"
-          :title="team.name"
-          :color="team.color"
-          :icon="team.logo"
-          :items="['Kacper', 'Laura', 'Krzysiek']"
-        ></btn-with-dropdown>
+        <btn-with-dropdown v-for="team of teams" :key="team.teamId" :title="team.name" :color="team.color"
+          :icon="team.logo" :items="['Kacper', 'Laura', 'Krzysiek']"></btn-with-dropdown>
```

Comment:
wiem ze to juz tak bylo ale jakbys widzial gdzies tagi które są w środku jak tutaj to mozesz przy okazji poprawiac je w ten sposób: 
```suggestion
          :icon="team.logo" :items="['Kacper', 'Laura', 'Krzysiek']"/>
```
ułatwia to czytelnosc i oszczedza sie miejsca